web/elm/src/Application/Application.elm

```diff
@@ -115,6 +115,18 @@ handleCallback callback model =
         OutputOfFetched (Err err) ->
             ( model, redirectToLoginIfNecessary err )
 
+        PipelineToggled _ (Err err) ->
+            subpageHandleCallback model callback
+                |> Tuple.mapSecond ((++) (redirectToLoginIfNecessary err))
```

Comment:
We do this in a bunch of other places already, don't we? Maybe it would make sense to include the `Tuple.mapSecond (++)` in the `redirectToLoginIfNecessary` function?

Reply:
d64a910e14841267a55fa89a239a4af35cd2d259

Reply:
LGTM!